components/feature/p2p/src/main/java/mozilla/components/feature/p2p/P2PFeature.kt

```diff
@@ -132,9 +132,16 @@ class P2PFeature(
             return
         }
 
-        val engineSession = sessionManager.getOrCreateEngineSession(session)
-        val messageHandler = P2PContentMessageHandler()
-        extensionController?.registerContentMessageHandler(engineSession, messageHandler)
+        // TODO Action for registering content message handler? Do we really need that?
```

Comment:
Let's look into this quickly before landing and file a follow-up if we can't address now.

Reply:
Updated the `P2PFeature` so it doesn't use `SessionManager` anymore and fixed the TODOs.